src/app/view/components/_common/action-button/action-button.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, Input, ViewEncapsulation } from '@angular/core';
+
+@Component({
+    encapsulation: ViewEncapsulation.None,
+    selector: 'action-button',
+    styles: [require('./action-button.scss')],
+    templateUrl: 'action-button.html'
+})
+
+export class ActionButtonComponent {
+    @Input() options: any[];
+    @Input() primaryAction: Function;
+    @Input() multipleOptions: Boolean;
+
+    ngOnInit(): void {
+        if (this.options === null) {
+            this.multipleOptions = false;
+        } else {
+            this.multipleOptions = true;
+        }
```

Comment:
All this can be just:

```this.multipleOptions = this.options !== null;```

But I don't think you need this, you can add the class by doing the check directly in the template, you can use ```options.length```